src/OSPSuite.Infrastructure.Import/Core/ColumnInfo.cs

```diff
@@ -57,5 +59,10 @@ public ColumnInfoCache(IReadOnlyList<ColumnInfo> columnInfos) : this()
       {
          AddRange(columnInfos);
       }
+
+      public IEnumerable<ColumnInfo> RelatedColumnsFrom(string measurement)
```

Comment:
I think the parameter should be a ColumnInof here. you always call it using column.Name. 
And it also reads better: RelatedColumnFrom (anotherColumn)

Reply:
> I think the parameter should be a ColumnInof here.

Not really so. For example in ColumnMappingPresenter.cs#L168, column is not a ColumnInfo but a Column (although Column also has a string Name getter). I do like renaming the parameter to anotherColumn though.